Reference/Management/Models/Language.md

```diff
@@ -22,6 +19,61 @@ All samples in this document will require the following using statement:
 
 ## Constructors
 
+### new Language(string isoCode)
+
+Constructor for creating a new `Language` object where the necessary parameter is a isoCode as a `string`.
+
 ## Properties
 
-## Methods
\ No newline at end of file
+### .CultureInfo
+
+Gets the CultureInfo object for the language.
+
+```csharp
+var language = new Language("en-US");
+return language.CultureInfo;
+```
+
+### .CultureName
+
+Gets or sets the culture name of the language.
+
+```csharp
+var language = new Language("en-US");
+return language.CultureName;
+```
+
+### .FallbackLanguageId
+
+Gets or sets the identifier of a fallback language. The fallback language can be used in multi-lingual scenarios, to help  define fallback strategies when a value does not exist for a requested language.
+
+```csharp
+var language = new Language("en-US");
+return language.FallbackLanguageId;
+```
+
+### .IsDefault
+Gets or sets a value indicating whether the language is the default language.
+
+```csharp
+var language = new Language("en-US");
+return language.IsDefault;
+```
+
+### .IsMandatory
+
+Gets or sets the identifier of a fallback language. The fallback language can be used in multi-lingual scenarios, to help define fallback strategies when a value does not exist for a requested language.
```

Comment:
I think this sentences is the same with .FallbackLanguageId was it intentional?

Reply:
@busrasengul  Yes your right. I updated the description.